File: video/getting-started/handle-participants/handle-participants.java
// Connect to room
Room room = Video.connect(context, connectOptions, new Room.Listener() {
    @Override
    public void onConnected(Room room) {}

    @Override
    public void onConnectFailure(Room room, TwilioException e) {}

    @Override
    public void onDisconnected(Room room, TwilioException e) {}

    @Override
    public void onRecordingStarted(Room room) {}

    @Override
    public void onRecordingStopped(Room room) {}

    @Override
    public void onParticipantConnected(Room room, Participant participant) {
        Log.i("Room.Listener", participant.getIdentity() + " has joined the room.");
    }

    @Override
    public void onParticipantDisconnected(Room room, Participant participant) {
        Log.i("Room.Listener", participant.getIdentity() + " has left the room.");
    }
);

// ... Assume we have received the connected callback

// After receiving the connected callback the LocalParticipant becomes available
LocalParticipant localParticipant = room.getLocalParticipant();
Log.i("LocalParticipant ", localParticipant.getIdentity());

// Get a participant from the room (let's assume we have a participant named Alice)
Participant participant = room.getParticipants().get("Alice");
Log.i("HandleParticipants", participant.getIdentity() + " is in the room.");

File: video/getting-started/handle-participants/handle-participants.js
// Log your Client's LocalParticipant in the Room
const localParticipant = room.localParticipant;
console.log(
  'Connected to the Room as LocalParticipant "%s"',
  localParticipant.identity
);

// Log any Participants already connected to the Room
room.participants.forEach(function(participant) {
  console.log(
    'Participant "%s" is connected to the Room',
    participant.identity
  );
});

// Log new Participants as they connect to the Room
room.once('participantConnected', function(participant) {
  console.log(
    'Participant "%s" has connected to the Room',
    participant.identity
  );
});

// Log Participants as they disconnect from the Room
room.once('participantDisconnected', function(participant) {
  console.log(
    'Participant "%s" has disconnected from Room',
    participant.identity
  );
});
